Content.Shared/Clothing/Components/ChameleonClothingComponent.cs

```diff
@@ -55,7 +55,7 @@ public ChameleonBoundUserInterfaceState(SlotFlags slot, string? selectedId)
 }
 
 [Serializable, NetSerializable]
-public sealed class ChameleonPrototypeSelectedMessage: BoundUserInterfaceMessage
+public sealed class ChameleonPrototypeSelectedMessage : BoundUserInterfaceMessage
```

Comment:
Whoops, didn't mean to include this change. Accidentally saved one tiny change while looking at Chameleon code to learn UI, and Github's been bugging me with it ever since.

Reply:
Just delete the extra space and push, it'll revert and remove it.

Reply:
Took a look at master, and it's actually a typo. It doesn't follow the rest of the formatting in the file. Mind if I keep my change?

Reply:
I don't see the harm in it, might as well keep it.